src/lib/confirm-bar.js

```diff
@@ -42,7 +42,7 @@ chrome.runtime.onMessage.addListener(function({message, iconData, closeData}) {
   }
   confirmationBar.outerHTML = `<div id="${confirmationId}" dir="${langDir}">
     <style scoped>
```

Comment:
Hmm, yeah, seeing in a few places ([like MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style#Attributes)) that scoped was deprecated awhile ago and is now disappearing from browsers 😞